components/Assessment/HackerEntry.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
```

Comment:
As this is a new component for Evaluator, can we move it under the `components/Evaluator` directory?

Reply:
Thanks did not notice the new directory!